pkg/pipeline/stream/message.go

```diff
@@ -54,6 +54,9 @@ type Message struct {
 	// Record represents a single record attached to the message.
 	Record record.Record
 
+	// SourceID contains the source connector ID.
+	SourceID string
```

Comment:
Source ID was moved from the record into the message, since it's an internal Conduit field, not something the record should be concerned with. This field is set in `SourceNode` when it creates the message.